packages/components/stencil-angularjs-output-target/rollup.config.js

```diff
@@ -0,0 +1,18 @@
+import pkg from './package.json';
+
+export default {
+  input: 'dist/index.js',
+  external: ['path', 'node-sass', 'fs', 'util'],
```

Comment:
**question**: Theses are mandatory?
Because I'm having trouble to understand why native modules such `fs` need to be declared as `external`.